contracts/BandOracle.cdc

```diff
@@ -183,69 +279,152 @@ pub contract BandOracle {
         emit BandOracleSymbolsUpdated(symbols: symbolsRates.keys, relayerID: relayerID, requestID: requestID)
     }
 
+    /// Auxiliary private function for removing a stored symbol
     ///
+    /// @param symbol: The string representing the symbol to delete
     ///
     access(contract) fun removeSymbol (symbol: String) {
-        BandOracle.symbolsRefData[symbol] = nil
+        BandOracle.symbolsRefData.remove(key: symbol)
```

Comment:
Seems like this state change should be broadcast in an event considering updates are emitted elsewhere - any reason it's not?

Reply:
Yeah that makes a lot of sense thanks!